### PR TITLE
(trivial) governance cleanup

### PR DESCRIPTION
**Pull request by UdjinM6**
- (trailing) spaces (lots of them)
- names
- no "using namespace std;"
- few log and rpc messages adjusted
- remove unused
- use defined types instead of declaring via std::map etc again
- move few types and class members to private (the only potential logic changes for future code, does not break any current code afaik)

### CODE REVIEW

File: src/governance-classes.h [hunk 1]
```diff
@@ -44,35 +43,24 @@ std::vector<std::string> SplitBy(std::string strCommand, std::string strDelimit)
...
 
-CAmount ParsePaymentAmount(const std::string& strAmount);
-
 class CGovernanceTriggerManager
 {
     friend class CSuperblockManager;
     friend class CGovernanceManager;
 
-public: // Typedefs
+private:
```
Comment (tgflynn) [inline src/governance-classes.h]: I don't think there's any real reason to make typedef's private they don't control access to any data.

File: src/governance.h [hunk 2]
```diff
@@ -25,32 +25,29 @@
...
 // FOR SEEN MAP ARRAYS - GOVERNANCE OBJECTS AND VOTES
 static const int SEEN_OBJECT_IS_VALID = 0;
 static const int SEEN_OBJECT_ERROR_INVALID = 1;
 static const int SEEN_OBJECT_ERROR_IMMATURE = 2;
 static const int SEEN_OBJECT_EXECUTED = 3; //used for triggers
 static const int SEEN_OBJECT_UNKNOWN = 4; // the default
 
+extern std::map<uint256, int64_t> mapAskedForGovernanceObject;
+extern CGovernanceManager governance;
+extern CCriticalSection cs_budget;
```
Comment (tgflynn) [inline src/governance.h]: Where did this come from ?  I had gotten rid of cs_budget and it's not present in the current 12.1 HEAD sources anywhere.

File: src/governance.h [hunk 3]
```diff
@@ -99,7 +96,7 @@ class CGovernanceManager
     // keep track of the scanning errors
     object_m_t mapObjects;
 
-    // note: move to private for better encapsulation 
+    // note: move to private for better encapsulation
```
Comment (tgflynn) [inline src/governance.h]: Since these are actually private now this comment should probably be removed.

**Comment by tgflynn** [comment]
Please remove the extraneous cs_budget, otherwise utACK.

File: src/governance.h (hunk 2 above)
Comment (UdjinM6) [inline src/governance.h]: ooops, way too old set of changes :) fixed.

File: src/governance.h (hunk 3 above)
Comment (UdjinM6) [inline src/governance.h]: fixed

**Comment by UdjinM6** [comment]
Thanks for the review @tgflynn 👍

**Pull request merged**